test/tests/recognizePDFTest.js

```diff
@@ -1,4 +1,4 @@
-describe.skip("PDF Recognition", function() {
+describe("PDF Recognition", function() {
```

Comment:
Yeah, it would still fail an assertion for the item existing in the collection, but the recognizePDF code should obviously send errors back up the chain. (That's specific to that code, not a general problem.) I'll deal with that separately as well. Thanks.